contracts/libraries/MathUtilsV2.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.5.11;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+library MathUtils {
+    using SafeMath for uint256;
+
+    // Divisor used for representing percentages
+    uint256 public constant PERC_DIVISOR = 1000000000;
```

Comment:
If we were to create a constant getter for this we could use that getter in the solidity unit tests instead of hardcoding the value but this is quite a trivial change so leaving that up to you.

e.g. 

```
function percDivisor() view returns (uint256) { return PERC_DIVISOR; }
```

```
    function test_validPerc() public {
        Assert.equal(MathUtils.validPerc(50), true, "50 should be a valid percentage");
        Assert.equal(MathUtils.validPerc(0), true, "0 should be a valid percentage");
        Assert.equal(MathUtils.validPerc(MathUtils.percDivisor()), true, "the max should be a valid percentage");
        Assert.equal(MathUtils.validPerc(MathUtils.percDivisor()+1), false, "1 more than the max should not be valid percentage");
    }
```